deepprofiler/dataset/compression.py

```diff
@@ -11,7 +11,7 @@
 import deepprofiler.dataset.image_dataset
 
 def png_dir(output_dir, plate_name):
-    return output_dir + "/" + plate_name + "/pngs/"
+    return output_dir + "/" + plate_name + "/"
```

Comment:
os.path.join() can be used

Reply:
Also, make sure the compressed images have their own directory and don't overwrite the originals